packages/dashboard/src/components/sidePanel/sections/aggregationSection/helpers.tsx

```diff
@@ -0,0 +1,40 @@
+import type { SelectProps } from '@cloudscape-design/components';
+import { AggregateType } from '@aws-sdk/client-iotsitewise';
```

Comment:
Nit: Should be type import?

Reply:
its actually the enum that we're importing in
<img width="446" alt="image" src="https://github.com/awslabs/iot-app-kit/assets/28601414/928fa177-c467-4822-9c1c-f41b6dc9c974">